src/Samples/SubModelSeq/Program.fs

```diff
@@ -13,177 +31,199 @@ module List =
         | a when a = j -> i
         | a -> a)
 
+  let swapWithNext i = swap i (i + 1)
+  let swapWithPrev i = swap i (i - 1)
+
+  let cons a ma = a :: ma
 
+        
 [<AutoOpen>]
-module Domain =
+module IdentifiableTypes =
+
+  type Identifiable<'a> =
+    { Id: Guid
+      Value: 'a }
```

Comment:
Is there a real benefit to this type? Why not just have each counter have an ID? (I have no IDE at the moment, only GH, so I may be missing something.)

Reply:
A single `Guid` is what's left after extracting all the single `Counter` fields to a type that is identical to the one in the `SingleCounter` sample.

Reply:
Fair enough. It's a balance between more reusable code (a general `Counter` has no ID) and simplicity (`Identifiable<_>` and its functions add some mental overhead).